demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
can we not use the metadata name as the pool name? That way we dont have to define another `name` arg in the spec.